Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/DiagnosticsTab/DiagnosticsStatus.tsx

```diff
@@ -49,42 +50,67 @@ export const DiagnosticsStatus = () => {
       css={{ height: '100%', display: 'flex', alignItems: 'center', paddingLeft: '8px' }}
       data-testid="diagnostics-tab-header--collapsed"
     >
-      <DefaultButton
-        ariaLabel={`${errorLabel} ${warningLabel}`}
-        styles={{
-          root: {
-            height: '36px',
-            padding: '0 5px',
-            minWidth: '20px',
-            border: 'none',
-            marginRight: '8px',
-          },
-          label: { fontSize: FontSizes.size18, fontFamily: 'Segoe UI', lineHeight: '20px' },
-        }}
-        onClick={() => {
-          setExpansion(true);
-          setActiveTab(DiagnosticsTabKey);
-        }}
-      >
-        {errorsCount > 0 && (
+      {errorsCount > 0 && (
+        <DefaultButton
+          ariaLabel={`${errorLabel}`}
+          styles={{
+            root: {
+              height: '36px',
+              padding: '0',
+              minWidth: '20px',
+              border: 'none',
+            },
+            label: { fontSize: FontSizes.size18, fontFamily: 'Segoe UI', lineHeight: '20px' },
+          }}
+          onClick={() => {
+            setErrorDiagnosticsFilter(true);
+            setWarningDiagnosticsFilter(false);
+            setExpansion(true);
+            setActiveTab(DiagnosticsTabKey);
+          }}
+        >
           <span css={{ marginRight: '10px', display: 'flex', alignItems: 'center' }}>
             <FontIcon
               css={{ color: SharedColors.red10, fontSize: FontSizes.size18, lineHeight: '18px', marginRight: '5px' }}
               iconName="StatusErrorFull"
             />
             {errorsCount}
           </span>
-        )}
-        {warningsCount > 0 && (
+        </DefaultButton>
+      )}
+
+      {warningsCount > 0 && (
+        <DefaultButton
+          ariaLabel={`${warningLabel}`}
+          styles={{
+            root: {
+              height: '36px',
+              minWidth: '20px',
+              border: 'none',
+            },
+            label: { fontSize: FontSizes.size18, fontFamily: 'Segoe UI', lineHeight: '20px' },
+          }}
+          onClick={() => {
+            setWarningDiagnosticsFilter(true);
+            setErrorDiagnosticsFilter(false);
+            setExpansion(true);
+            setActiveTab(DiagnosticsTabKey);
+          }}
+        >
           <span css={{ display: 'flex', alignItems: 'center' }}>
             <FontIcon
-              css={{ color: SharedColors.yellow10, fontSize: FontSizes.size18, lineHeight: '18px', marginRight: '5px' }}
+              css={{
+                color: `#F4BD00`,
```

Comment:
avoid hex colors

Reply:
Yep. Was planning to use the Fluent palette yellow